### PR TITLE
fix(message): update timestamp message every minute

### PR DESCRIPTION
**Pull request by vimercati-samir**
**What this PR does** 📖
Update timestamp message every minute

**Which issue(s) this PR fixes** 🔨
timestamp message not updating (‘A few seconds ago' for ever unless component rebuilds)

Fixes #AP-43

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by stavares843 — APPROVED** [review]
so it no longer gets stuck showing a few seconds ago 


Captura de ecrã 2021-11-23, às 18 19 19

when sending a message says a few seconds ago and then updates to a minute ago, 2 minutes ago, and so on

but the thing is, when you send a message and then wait 5 minutes and send another message, all those messages shows that were sent 5 minutes ago, even if the oldest one was 5m ago and the newest one was a couple of seconds ago

I'm gonna approve this because that's probably to be handled on other PR, let me know cc @WanderingHogan

**Comment by WanderingHogan** [comment]
@stavares843 just added a few tickets to address what you brought up: AP-106 and AP-107

**Review by WanderingHogan — APPROVED** [review]
great idea with clearInterval when you tear down the component

File: components/tailored/messaging/group/Group.vue [hunk 1]
```diff
@@ -34,6 +40,17 @@ export default Vue.extend({
       this.$store.commit('ui/setQuickProfilePosition', e)
       this.$store.commit('ui/quickProfile', true)
     },
+    refreshTimestampEveryMinute() {
+      this.timestampRefreshInterval = setInterval(() => {
+        this.$data.timestamp = this.$dayjs(this.group.at).from()
+      }, 60 * 1000)
```
Comment (RetroPronghorn) [inline components/tailored/messaging/group/Group.vue:46]: Can we move this to a config value?

File: components/tailored/messaging/message/Message.vue [hunk 2]
```diff
@@ -179,6 +181,17 @@ export default Vue.extend({
         from: this.$props.group.id,
       })
     },
+    refreshTimestampEveryMinute() {
+      this.timestampRefreshInterval = setInterval(() => {
+        this.$data.timestamp = this.$dayjs(this.$props.message.at).from()
+      }, 60 * 1000)
```
Comment (RetroPronghorn) [inline components/tailored/messaging/message/Message.vue:187]: Config value

**Review by RetroPronghorn — CHANGES_REQUESTED** [review]

File: components/tailored/messaging/message/Message.vue (hunk 2 above)
Comment (RetroPronghorn) [reply components/tailored/messaging/message/Message.vue:187]: I suggest 
```ts
 chat: {
    defaultMessageLimit: 50,
    defalutLoadMoreLimit: 20,
    timestampUpdateInterval: 60 * 1000,
  },
  ```

**Review by RetroPronghorn — APPROVED** [review]

**Pull request merged**